tools/fuchsia/fuchsia_archive.gni

```diff
@@ -67,12 +67,37 @@ template("package_dir") {
     ]
   }
 
-  action(target_name) {
+  pkg_dir_deps = pkg.deps + [ ":$cmx_target" ]
+
+  action("${target_name}_dir") {
     script = "$flutter_root/tools/fuchsia/copy_path.py"
     response_file_contents = rebase_path(copy_sources + copy_outputs)
-    deps = pkg.deps + [ ":$cmx_target" ]
+    deps = pkg_dir_deps
     args = [ "--file-list={{response_file_name}}" ]
     outputs = copy_outputs
     testonly = pkg_testonly
   }
+
+  action(target_name) {
+    script = "$flutter_root/tools/fuchsia/gen_package.py"
+    deps = pkg_dir_deps + [ ":${target_name}_dir" ]
+    sources = copy_outputs
+    inputs = [
+      "$flutter_root/tools/fuchsia/gather_flutter_runner_artifacts.py",
```

Comment:
shouldn't inputs be the same as inputs for `package_dir`

Reply:
This was neeeded because the gen_package script imported that.  It's not needed anymore.